docs/api/cases/cases-api-add-comment.asciidoc

```diff
@@ -39,15 +39,20 @@ default space is used.
 === {api-request-body-title}
 
 `alertId`::
-(Required*, string or array of strings) The alert identifier. It is required
-only when `type` is `alert`. If it is an array, `index` must also be an array. preview:[]
+(Required*, string or array of strings) The alert identifiers. It is required
```

Comment:
In the `rule` description we say "The rule that is associated with the alert". Should we change it to say `or alerts`?

Reply:
This seems to indicate that though you can add multiple alerts to a case in a single API call, they must all be related to the same rule. Is that correct?

Reply:
Thanks, I've updated the text again via 040931b9b4b93812f6d7aef508ff323f2ab51886